service/resource/deployment/current.go

```diff
@@ -49,20 +47,35 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 	return currentDeployments, nil
 }
 
-func getVersionBundleVersionInfos(labels map[string]string) (string, string, string, error) {
-	version, ok := labels[VersionBundleVersionLabel]
-	if !ok {
-		return "", "", "", microerror.Maskf(executionFailedError, "label '%s' must not be empty", VersionBundleVersionLabel)
-	}
+func updateVersionBundleVersionGauge(gauge *prometheus.GaugeVec, deployments []*v1beta1.Deployment) error {
```

Comment:
I wouldn't bother passing the gauge in

Reply:
My thinking was towards testability. It is maybe easier to test the method that way when it does not depend on magic global vars that much.

Reply:
No strong opinion <£